steps/etcd/etcd.go

```diff
@@ -67,6 +67,23 @@ func Setup(fsc *fs.FsClient, sc *systemd.SystemdClient, fc fetchclient.FetchClie
 	return nil
 }
 
+func createEtcdService(fsc *fs.FsClient, useOverlay bool) error {
```

Comment:
Aligned the way how the service template is created with how it is done in the docker step.